dangerzone/isolation_provider/base.py

```diff
@@ -93,8 +93,82 @@ def convert(
             self.print_progress(document, True, str(e), 0)
             document.mark_as_failed()
 
-    def doc_to_pixels(
-        self, document: Document, tempdir: str, p: subprocess.Popen
+    def ocr_page(self, pixmap: fitz.Pixmap, ocr_lang: str) -> bytes:
+        """Get a single page as pixels, OCR it, and return a PDF as bytes.
+
+        This operation is particularly tricky, since we have to handle various PyMuPDF
+        versions.
+        """
+        if int(fitz.version[2]) >= 20230621000001:
```

Comment:
nitpick: the `int()` call might be superfluous. Should we directly compare strings instead?

Reply:
Closing this thread, since this block of code has been removed.